README.md

```diff
@@ -32,6 +32,63 @@ bundle out of these asset-feeds and persist this. It will respond with the URL t
 
 This client helps with remotely triggering these steps in the [asset-pipe-build-server][asset-pipe-build-server].
 
+## Optimistic Bundling
+
+A new approach to bundling assets has been added which we are calling optimistic bundling. This feature allows for assets to react to changes in uploaded assets automatically.
+
+This works in the following way:
+
+1. Any number of assets are uploaded to the asset server using the `client.publishAssets` method.
+2. Bundling instructions are uploaded to the asset server using the `client.publishInstructions` method.
+
+Steps 1 and 2 can be performed in any order and the asset server will automatically bundle and rebundle as needed to produce up to date bundles for any instructions that have been published to the server.
+
+**Examples**
+
+We might begin by first we publishing some bundling instructions. (Though we could just as easily publish assets first):
+
+```js
+const { success } = await client.publishInstructions('layout', 'js', ['podlet1', 'podlet2']);
```

Comment:
`success`? Shouldn't the call reject if not?

Reply:
yea, it rejects on failure. Success property is only present on success... not very nice is it. Perhaps i'll just change to a 204 with no content?